pypesto/objective/amici_objective.py

```diff
@@ -313,10 +323,9 @@ def _call_amici(
             # set parameters in model, according to mapping
             self.set_par_sim_for_condition(data_ix, x)
 
-            if self.preequilibration_edatas:
-                original_value_dict = self.preprocess_preequilibration(data_ix)
-            else:
-                original_value_dict = None
+            if 'fval' in self.preeq_guesses and \
```

Comment:
this double usage of fval looks a bit strange. maybe test for ``self.guess_steadystate``?